draft-ietf-quic-transport.md

```diff
@@ -6387,6 +6387,36 @@ following properties:
 Note that these guarantees are the same guarantees provided for any NAT, for the
 same reasons.
 
+## Considerations for 5-tuple routing architectures
+
+QUIC servers can be deployed behind a 5-tuple based routing architecture that
+delivers packets based on both the source and destination IP addresses and
+ports. In such an architecture, clients that change IP address or port are
+likely to be routed to a different server. There are several actions that can
+mitigate or resolve operational and security issues in this case.
+
+* Servers can use an out-of-band mechanism to deliver packets to the correct
+destination or transfer state from the original destination.
+
+* If the server has another address where the 5-tuple based routers are not on-
+path, the preferred_address transport parameter can communicate that address and
+thus support changing client IP addresses without difficulty.
```

Comment:
The text sounds a bit too affirmative to me.

Not all clients would support the preferred_address TP, even those supporting the TP might have reasons not to migrate to the preferred address. All the server can do is *hope* that they would migrate.

Reply:
You're right. I forgot response to preferred_address is a SHOULD. I'll fix it tomorrow.